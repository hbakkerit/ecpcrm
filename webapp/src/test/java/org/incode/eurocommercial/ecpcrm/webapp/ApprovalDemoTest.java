package org.incode.eurocommercial.ecpcrm.webapp;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.junit.Test;

@UseReporter(MyBeyondCompare3Reporter.class)
public class ApprovalDemoTest {


    @Test
    public void testBasicFormatting() throws Exception
    {
        String json = "{\"infos\":{\"address\":\"my address\",\"phone\":\"my phone\"},\"insurance\":{\"forks\":[14,53,123],\"prices\":[5,8,\"3%\"]}}";
        Approvals.verifyJson(json);
    }


}
