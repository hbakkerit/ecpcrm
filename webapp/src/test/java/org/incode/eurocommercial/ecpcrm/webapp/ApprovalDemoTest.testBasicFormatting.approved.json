{
  "infos": {
    "address": "my address",
    "phone": "my phone"
  },
  "insurance": {
    "forks": [
      14,
      53,
      123
    ],
    "prices": [
      5,
      8,
      "3%"
    ]
  }
}